### PR TITLE
Feat/connects-form-req-to-backend

### PR DESCRIPTION
**Pull request by ChristianAlexa**

### CODE REVIEW

File: server/controllers/requests.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ export default class RequestsController {
       });
   }
 
-  @Post()
+  @Post('/create')
```
Comment (ConnorDY) [inline server/controllers/requests.ts:54]: Let's keep this RESTful and undo this change.

File: src/views/NewRequestView.tsx [hunk 2]
```diff
@@ -38,27 +38,27 @@ const NewRequestView: React.FC<{ user: User | undefined }> = ({ user }) => {
...
+      const data = {
+        maskShieldCount,
+        details,
+        jobRole,
+        email,
+        facilityName,
+        addressCity,
+        addressState,
+        addressZip,
+        status: 'requested',
```
Comment (ConnorDY) [inline src/views/NewRequestView.tsx:50]: I think the API should set this so this UI shouldn't have to provide it.

Comment (ConnorDY) [reply src/views/NewRequestView.tsx:50]: Additionally, we are currently expecting it to be stored in Title Case (ie `Requested`) so that it can easily be displayed back without needing to manipulate the string.

File: src/views/NewRequestView.tsx [hunk 3]
```diff
@@ -38,27 +38,27 @@ const NewRequestView: React.FC<{ user: User | undefined }> = ({ user }) => {
...
-            console.error(err);
-          });
-      }
+      axios
+        .post(buildEndpointUrl('requests/create'), data)
+        .then((res) => {
+          setIsCreated(!isCreated);
+        })
+        .catch((err) => {
+          console.error(err);
```
Comment (ConnorDY) [inline src/views/NewRequestView.tsx:60]: **TODO:**
Add `react-toastify` and display an error toast here.

Comment (mjoyce91) [reply src/views/NewRequestView.tsx:60]: yasss `react-toastify` === 💯
